dojo/benchmark/views.py

```diff
@@ -130,78 +192,140 @@ def score_asvs(product, benchmark_type):
     benchmark_product_summary.save()
 
 
-@user_is_authorized(Product, Permissions.Benchmark_Edit, 'pid')
+@user_is_authorized(Product, Permissions.Benchmark_Edit, "pid")
 def benchmark_view(request, pid, type, cat=None):
     product = get_object_or_404(Product, id=pid)
     benchmark_type = get_object_or_404(Benchmark_Type, id=type)
-    benchmark_category = Benchmark_Category.objects.filter(type=type, enabled=True).order_by('name')
+    benchmark_category = Benchmark_Category.objects.filter(
+        type=type, enabled=True
+    ).order_by("name")
 
     # Add requirements to the product
-    new_benchmarks = Benchmark_Requirement.objects.filter(category__type=type, category__type__enabled=True,
-                                                          enabled=True).exclude(
-        id__in=Benchmark_Product.objects.filter(product=product).values_list('control_id', flat=True))
+    new_benchmarks = Benchmark_Requirement.objects.filter(
+        category__type=type, category__type__enabled=True, enabled=True
+    ).exclude(
+        id__in=Benchmark_Product.objects.filter(product=product).values_list(
+            "control_id", flat=True
+        )
+    )
     add_benchmark(new_benchmarks, product)
 
     # Create the benchmark summary category
     try:
-        benchmark_product_summary = Benchmark_Product_Summary.objects.get(product=product, benchmark_type=benchmark_type)
-    except:
-        benchmark_product_summary = Benchmark_Product_Summary(product=product, benchmark_type=benchmark_type)
+        benchmark_product_summary = Benchmark_Product_Summary.objects.get(
+            product=product, benchmark_type=benchmark_type
+        )
+    except BaseException:
```

Comment:
```suggestion
    except Exception:
```

Reply:
Changes apply in commit https://github.com/DefectDojo/django-DefectDojo/pull/8321/commits/09e8746cdb2cce9fd1e93630862bc819b95e5c15